Scheduling/EventSourcing/EventMetadata.cs

```diff
@@ -1,7 +1,14 @@
+using Newtonsoft.Json;
+
 namespace Scheduling.EventSourcing;
 
 public record EventMetadata(
-    string ClrType,
     CorrelationId CorrelationId,
-    CausationId CausationId
-);
\ No newline at end of file
+    CausationId CausationId,
+    [property: JsonIgnore] 
```

Comment:
**COMMENT:** This is ignored, as we don't need to store this information in EventMetadata. We have it inside the resolved event data.